SubjuGator/command/subjugator_launch/config/adaptive_controller.yaml

```diff
@@ -1,6 +1,7 @@
 ---
-kp: 1000, 1000, 1000, 5000, 5000, 5000
-kd: 150, 150, 150, 50, 100, 25
-kg: 5,5,5,5,5,5
-ki: 5,5,5,5,10,5
-use_learned: false
+kp: 120, 150, 200, 100, 50, 50
+kd: 25, 25, 25, 25, 25, 25
+#kd: 0, 0, 0, 0, 0, 0
+kg: 2.5,2.5,2.5,2.5,2.5,2.5
+ki: 2.5,2.5,2.5,2.5,2.5,2.5
+use_learned: true
```

Comment:
I'm not going to approve this until we have a way to differentiate between PID gains for simulation vs. real-life. But, this is a good start!

We would just need to update this file with _both_ values, and have a parameter to use one set of values vs. the other set.

Reply:
We would probably need some ROS parameter or argument that we can reference in `sub8.launch` and `gazebo.launch` for the adaptive controller correct? I'll see if I can reference some files with this type of branching.